src/Microsoft.Identity.Client/TokenCache.ITokenCacheInternal.cs

```diff
@@ -134,7 +134,8 @@ public sealed partial class TokenCache : ITokenCacheInternal
                             msalRefreshTokenCacheItem = new MsalRefreshTokenCacheItem(
                                 preferredEnvironmentHost,
                                 requestParams.ClientId,
-                                response);
+                                response,
+                                subject);
```

Comment:
I don't understand this change. Is it related to the fix?

Reply:
Yes, it resolves the refresh token cache lookup error